src/lib/config.ts

```diff
@@ -1059,8 +1060,23 @@ export class default_config {
     visualexitauto: "true" | "false" = "true"
 }
 
+const windows_defaults = {
+    browsermaps: {
+        "<C-6>": null,
+        "<CS-6>": null,
+        "<A-6>": "buffer #",
+        "<AS-6>": "buffer #",
+    } as unknown, // typescript doesn't like us adding new binds like this
+} as default_config
+
+
+/** @hidden
+ * Merges two objects and removes all keys with null values at all levels
+ */
+export const mergeDeepCull = R.pipe(mergeDeep, removeNull)
+
 /** @hidden */
-export const DEFAULTS = o(new default_config())
+export const DEFAULTS = platform.getPlatformOs() == "win" ? mergeDeepCull(o(new default_config()), windows_defaults) : o(new default_config())
```

Comment:
Should probably make this more extensible. The truly sane thing to do would be to have a `set --platform` setting like `set --url` (i.e. `:seturl` as it actually exists), but I'm not sure I can be bothered.